frontend/src/libs/hooks/use-sidebar-show/use-sidebar-show.hook.ts

```diff
@@ -0,0 +1,5 @@
+const useSidebarShow = (searchParameter: string | null): boolean => {
```

Comment:
we should call useSearchParams in this hook, let's rename it to useSidebarState and return isSidebarShown and setIsSidebarShown from it
```suggestion
const useSidebarState = (searchParameter: string | null): boolean => {
```

Reply:
done